dashboard/src/shared/PackagesService.ts

```diff
@@ -17,14 +17,35 @@ export default class PackagesService {
     size: number,
     query?: string,
   ): Promise<GetAvailablePackageSummariesResponse> {
-    return await this.client().GetAvailablePackageSummaries({
+    let request = {
       context: { cluster: cluster, namespace: namespace },
-      filterOptions: {
-        query: query,
-        repositories: repos.split(","),
-      },
       paginationOptions: { pageSize: size, pageToken: page.toString() },
-    });
+    };
+
+    // Add filter options only as needed.
```

Comment:
Great, thanks for this change. I assumed it was ok with passing `""` and `[""]` to the plugin, so I didn't care about it so much.